### PR TITLE
Add Job Summary

### PR DESCRIPTION
**Pull request by bradyclifford**
# Summary of PR changes

Some tests are ran as a runbook, not within a PR.  Status checks and PR comments will not work.  Instead a Job Summary is created.

## PR Requirements
- [ ] For major, minor, or breaking changes, at least one of the commit messages contains the appropriate `+semver:` keywords.
  - See the *Incrementing the Version* section of the repository's README.md for more details.
- [x] The action code does not contain sensitive information.

*NOTE: If the repo's workflow could not automatically update the `README.md`, it should be updated manually with the next version.  For javascript actions, if the repo's workflow could not automatically recompile the action it should also be updated manually as part of the PR.*

### CODE REVIEW

**Comment by bradyclifford** [comment]
If this change it not desired, I will consider asking the Infra Purple team to add the action https://github.com/marketplace/actions/publish-test-results to im-open.

**Comment by bradyclifford** [comment]
Looking for reviews.  Once the pattern is agreed on, I will build the bundle and include any other missing items.

File: src/github.js [hunk 1]
```diff
@@ -111,6 +111,10 @@ async function createPrComment(repoToken, markupData, updateCommentIfOneExists,
   }
 }
 
+async function createJobSummary(markupData) {
```
Comment (hpractv) [inline src/github.js:114]: Where is this getting called in case the Summary is desired?

Comment (bradyclifford) [reply src/github.js:114]: It's not a complete PR.  Just showing the concept.  Once I get a thumbs up that your team is ok with the additional, I will finish up the missing pieces.

I didn't want to waste any cycles on something the BC swat team didn't even want implemented.

Comment (danielle-casella-adams) [reply src/github.js:114]: @bradyclifford I think we're fine with adding the summary to this action.  Is this something you wanted to continue with or should we close this one out if you'll be moving to the other action?

Comment (bradyclifford) [reply src/github.js:114]: We can close this PR. I am going to use the new action instead now.

**Pull request closed without merging**